pkg/slo-controller/noderesource/plugins/cpunormalization/plugin.go

```diff
@@ -176,8 +176,14 @@ func (p *Plugin) Calculate(_ *configuration.ColocationStrategy, node *corev1.Nod
 
 	ratio, err := getCPUNormalizationRatioFromModel(basicInfo, strategy)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get ratio in cpu normalization calculation, err: %s", err)
+		if strategy.DefaultRatio != nil {
```

Comment:
recommend to move the default logic inside the function getCPUNormalizationRatio